modules/src/error_msg.py

```diff
@@ -0,0 +1,55 @@
+
```

Comment:
Let's make only one file now: examples.py. (error_msg.py can be created later if required; let the error messages be in the corresponding module itself, like it is now. Will visit it later)
examples file would have a list of sample queries for each of the modules, which will be used in both src and tests.